modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/example.css

```diff
@@ -0,0 +1,4 @@
+.frontend_stylesheet_example {
```

Comment:
Would the JSP be able to use style definitions created in another file being `@import`ed from here? I think so if the URL we generate is served by the provider bundle, just wanted to double check

Reply:
It should. 

If the `@import` is processed by Sass, everything is done at build time, so it's transparent (only one CSS file is generated and it contains everything it needs).

If the `@import` is processed by the browser, it will look for the imported CSS in the same base URL, so it will find it too.

The only caveat is that if you have -say- `a.css` which imports `b.css` (browser-side) and then you place `b.css` alone in a page, there will be two inclusions of `b.css`. But this is how the standard web model works, not anything Liferay specific. Also, the cache would do its magic, so we don't need to worry.